cocos2d/core/components/CCAudioSource.js

```diff
@@ -132,16 +132,13 @@ var AudioSource = cc.Class({
                     if (this._mute) {
                         if (CC_JSB) {
                             cc.audioEngine.setEffectsVolume(0);
-                        }
-                        else {
+                        } else {
```

Comment:
Makes sure these boxes are checked before submitting your PR - thank you!
- [ ] Check that your javascript is following our [style guide](https://github.com/cocos-creator/fireball/blob/dev/.github/CONTRIBUTING.md) ...